crates/driver/src/infra/api/error.rs

```diff
@@ -0,0 +1,104 @@
+use {
+    crate::{
+        domain::competition::{self, quote, solution},
+        infra::api,
+    },
+    hyper::StatusCode,
+    serde::Serialize,
+};
+
+#[derive(Debug, Clone, Copy, Serialize)]
+#[serde(rename_all = "PascalCase")]
+enum Type {
```

Comment:
Should this be called `Kind` to be more Rust-y?

Reply:
https://github.com/cowprotocol/services/pull/1128/commits/abda069a433854826d5d895ec5ddfbc901532527